Packs/CommonScripts/Scripts/CheckPDFEncryptionAndValidity/CheckPDFEncryptionAndValidity.yml

```diff
@@ -0,0 +1,36 @@
+args:
+- default: false
+  description: The entry ID of the PDF file.
+  isArray: false
+  name: EntryID
+  required: true
+comment: 'Returns if a PDF is valid and if it is encrypted.'
+commonfields:
+  id: CheckPDFEncryptionAndValidity
+  version: -1
+enabled: false
+name: CheckPDFEncryptionAndValidity
+outputs:
+- contextPath: File.IsEncrypted
+  description: 'True if PDF file is encrypted, False otherwise.'
+  type: Boolean
+- contextPath: File.IsValid
+  description: 'True if PDF file is readable, False otherwise.'
+  type: Boolean
+- contextPath: File.EntryID
+  description: 'The entry ID of the file.'
+  type: String
+script: '-'
+system: false
+tags:
+- basescript
+timeout: '0'
+type: python
+subtype: python3
+dockerimage: demisto/readpdf:1.0.0.2034953
+fromversion: 5.5.0
```

Comment:
Since its new I dont think it should be from 5.5, lets consult with sec team, I think we can assume much newer.